ansible/roles/package-upgrade/tasks/partials/yum.yml

```diff
@@ -5,6 +5,6 @@
 #
 
   - name: upgrade installed packages
-    yum: name=* state=latest use_backend=yum
+    yum: name=* state=latest use_backend=yum expire-cache=yes
```

Comment:
? `expire-cache` isn't an option on the `yum` task: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/yum_module.html
Perhaps the intent is `update_cache`?

Reply:
`expire-cache` is an alias to `update_cache`. I figured it was more readable since the yum command equivalent is `yum clean expire-cache`

Reply:
🤦 I missed the smallprint!